backend/controller/paymentController.js

```diff
@@ -140,8 +196,13 @@ const createPlan = asyncHandler(async (req, res, next) => {
   });
 
   if (planResponse.error) {
-    // FIXME: What if there is no error.description? Provide a custom message as well for double sure
-    return next(new CustomError(planResponse.error.description, 400));
+    return next(
+      new CustomError(
+        planResponse.error.description ||
+          "Error occurred during creating subscription plan",
+        400
```

Comment:
This is from Stripe, so make it a 5xx error since it is from our side and not user side

Reply:
done pass 500 status code